stacktrace_ext_test.go

```diff
@@ -120,6 +120,31 @@ func TestStacktraceFiltersVendorZap(t *testing.T) {
 	})
 }
 
+func TestStacktraceWithoutCallerSkip(t *testing.T) {
+	withLogger(t, func(logger *zap.Logger, out *bytes.Buffer) {
+		func() {
+			logger.Error("test log")
+		}()
+
+		require.Contains(t, out.String(), "TestStacktraceWithoutCallerSkip.", "Should not skip too much")
+		require.Contains(t, out.String(), "TestStacktraceWithoutCallerSkip", "Should not skip too much")
```

Comment:
If we pass the first check, we'll always also pass the second check. We can probably just drop the second.
```suggestion
		require.Contains(t, out.String(), "TestStacktraceWithoutCallerSkip.", "Should not skip too much")
```